proposed/http-message.md

```diff
@@ -85,7 +85,7 @@ convenience of working with headers as strings, headers can be retrieved from
 an instance of a ``MessageInterface`` as an array or string. Use the
 `getHeader()` method to retrieve a header value as a string containing all
 header values of a header by name concatenated with a comma.
-Use `getHeaderAsArray()` to retrieve an array of all the header values for a
+Use `getHeaderLines()` to retrieve an array of all the header values for a
```

Comment:
I don't like `getHeaderLines` since it's about getting multiple lines of the same header (probably a single one?)